src/consumer.cpp

```diff
@@ -134,6 +135,21 @@ void Consumer::unassign() {
     check_error(error);
 }
 
+void Consumer::incremental_assign(const TopicPartitionList& topic_partitions) {
+    if (topic_partitions.empty()) {
```

Comment:
Is this right? The [documentation](https://github.com/edenhill/librdkafka/blob/master/src/rdkafka.h#L3881) says:

>  The application must pass the partition list passed to the callback (or a copy of it), even if the list is empty.